core/server/types/app.go

```diff
@@ -26,9 +30,9 @@ func AppAddProtoToCustomResource(msg *pb.AddAppRequest) *v1alpha2.Application {
 			Name:      msg.Name,
 			Namespace: msg.Namespace,
 			Labels: map[string]string{
-				"app.kubernetes.io/part-of":    msg.Name,
-				"app.kubernetes.io/managed-by": "weave-gitops",
-				"app.kubernetes.io/created-by": "kustomize-controller",
+				PartOfLabel:    msg.Name,
+				ManagedByLabel: "weave-gitops",
```

Comment:
Probably we can replace `"weave-gitops"` with `managedByWeaveGitops`.

Reply:
Fixed